website/versioned_docs/version-v2.0_alpha/quick-start/poll-finalization.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Poll Finalization
+description: How to finalize a MACI Poll
+sidebar_label: Poll Finalization
+sidebar_position: 4
+---
+
+As a coordinator, first you need to merge signups and messages (votes). Messages are stored in a queue so when the poll in over, the coordinator needs to create the tree from the queue. This optimization is needed to reduce gas cost for voters. Then coordinator generates proofs for the message processing, and tally calculations. This allows to publish the poll results on-chain and then everyone can verify the results when the poll is over.
```

Comment:
nit: when the poll **is** over

And the `queue` here is the AccQueue, maybe could link to the contracts/accqueue page to remind user of how it works? (=why we call it an optimization)